diff --git a/build/lib/logging.js b/build/lib/logging.js
index f609feec3b89ef005177392a54c95560679630bf..d12007dd1880230b1900785daafb9225f697f114 100644
--- a/build/lib/logging.js
+++ b/build/lib/logging.js
@@ -13,7 +13,6 @@ exports.LEVELS = ['silly', 'verbose', 'debug', 'info', 'http', 'warn', 'error'];
 const MAX_LOG_RECORDS_COUNT = 3000;
 const PREFIX_TIMESTAMP_FORMAT = 'HH-mm-ss:SSS';
 // export a default logger with no prefix
-exports.log = getLogger();
 // mock log object is used in testing mode to silence the output
 const MOCK_LOG = {
     unwrap: () => ({
@@ -27,6 +26,7 @@ const MOCK_LOG = {
     }),
     ...(lodash_1.default.fromPairs(exports.LEVELS.map((l) => [l, lodash_1.default.noop]))),
 };
+exports.log = getLogger();
 /**
  *
  * @param {AppiumLoggerPrefix?} [prefix=null]
