diff --git a/build/lib/driver.d.ts b/build/lib/driver.d.ts
index e432a4f7fce958ab51f5b46bfab364928a8c972b..e0b61f0224b4bcbd69d86454485d5935ba0e7654 100644
--- a/build/lib/driver.d.ts
+++ b/build/lib/driver.d.ts
@@ -919,7 +919,7 @@ declare class AndroidUiautomator2Driver extends AndroidDriver implements Externa
     proxyActive(sessionId: string): boolean;
     canProxy(sessionId: string): boolean;
     getProxyAvoidList(): RouteMatcher[];
-    updateSettings(settings: Uiautomator2Settings): Promise<void>;
+    updateSettings(settings: StringRecord): Promise<void>;
     getSettings(): Promise<any>;
 }
 import './commands';
diff --git a/lib/driver.ts b/lib/driver.ts
index 531e29905a84210c7d57f695f57b7951d8eeccfc..deaac8be636b7069160907f0089252ab1ff347b1 100644
--- a/lib/driver.ts
+++ b/lib/driver.ts
@@ -874,7 +874,7 @@ class AndroidUiautomator2Driver
     return this.jwpProxyAvoid;
   }
 
-  async updateSettings(settings: Uiautomator2Settings) {
+  async updateSettings(settings: StringRecord) {
     await this.settings.update(settings);
     await this.uiautomator2!.jwproxy.command('/appium/settings', 'POST', {settings});
   }